eng/common/scripts/stress-testing/deploy-stress-tests.ps1

```diff
@@ -21,8 +21,8 @@ $ErrorActionPreference = 'Stop'
 $FailedCommands = New-Object Collections.Generic.List[hashtable]
 
 . (Join-Path $PSScriptRoot "../Helpers" PSModule-Helpers.ps1)
-Install-ModuleIfNotInstalled "powershell-yaml" "0.4.1" | Import-Module
-Install-ModuleIfNotInstalled "az" "6.4.0" | Import-Module
+Install-ModuleIfNotInstalled "powershell-yaml" "0.4.1" | Import-Module -ErrorAction Continue
```

Comment:
If this is a common error we should consider adding a helper function for importing.